osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSongSelect.cs

```diff
@@ -1,9 +1,13 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+#nullable enable
```

Comment:
Should be removed now after https://github.com/ppy/osu/pull/18743 (and is also the correct merge resolution).